actions/storage.js

```diff
@@ -31,12 +34,9 @@ export function removeItem(name) {
 }
 
 export function setGlobalItem(name, value) {
-    return (dispatch) => {
-        dispatch({
-            type: StorageTypes.SET_GLOBAL_ITEM,
-            data: {name, value, timestamp: new Date()},
-        });
-        return {data: true};
+    return {
+        type: StorageTypes.SET_GLOBAL_ITEM,
+        data: {name, value, timestamp: new Date()},
```

Comment:
🎉

Reply:
I wanted to do this to the rest of the action creators too, but I didn't feel like making more changes to this file at the moment 😛